tutorial/markdown/aspnet/semantic-kernel/semantic-kernel-tutorial.md

```diff
@@ -0,0 +1,388 @@
+---
+# frontmatter
+path: "/tutorial-csharp-semantic-kernel-vector-search"
+# title and description do not need to be added to markdown, start with H2 (##)
+title: Build Vector Search with Couchbase .NET Semantic Kernel Connector and OpenAI
+short_title: Vector Search with Semantic Kernel
+description:
+  - Build a semantic search application using Couchbase Hyperscale vector index with Semantic Kernel.
```

Comment:
Should it be both Couchbase Hyperscale vector index and Composite vector index or Vector Search?